projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -289,6 +289,22 @@ public function use_admin_ui_v1(): bool {
 			|| $this->has_connections_management_feature();
 	}
 
+	/**
+	 * Whether the user has the feature flag enabled.
+	 *
+	 * @param string $flag_name The feature flag to check. Will be prefixed with 'jetpack_social_use_' for the option.
+	 * @param string $plan_name The plan name to check for for the Current_Plan check, without the social- prefix.
+	 * @return bool
+	 */
+	public function use_feature_flag( $flag_name, $plan_name ): bool {
+		// If the option is set, use it.
+		if ( get_option( 'jetpack_social_use_' . $flag_name, false ) ) {
+			return true;
+		}
+
+		return Current_Plan::supports( 'social-' . $plan_name );
```

Comment:
Apart from this, having a constant defined for a feature to be enabled makes it easier than updating an option in the database.

Should we have something like this

```php
if ( defined( 'JETPACK_SOCIAL_HAS_' . strtoupper( $flag_name ) ) && constant( 'JETPACK_SOCIAL_HAS_' . strtoupper( $flag_name ) ) ) {
	return true;
}
```

Reply:
Use can also use `jetpack docker wp option update jetpack_social_use_editor_preview true` in the CLI but I can add that too sure 👍

Reply:
Yeah, that works for local sites but it's not easy for JN/WoA sites.